draft-ietf-tls-tls13.md

```diff
@@ -1670,7 +1674,7 @@ values, it is not possible for an active attacker to modify the
 randoms without detection as long as ephemeral ciphers are used.
 It does not provide downgrade protection when static RSA is used.
 
-Note: This is an update to TLS 1.2 so in practice many TLS 1.2 clients
+Note: This is a change from RFC 5246, so in practice many TLS 1.2 clients
```

Comment:
I waffled a bit about linkifying, but I guess it does make more sense to do so than not.